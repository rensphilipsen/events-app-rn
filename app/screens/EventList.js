import React, {PureComponent} from 'react';
import Container from '../components/Container/Container';
import EventListView from '../components/EventListView/EventListView';

class EventList extends PureComponent {

	static navigationOptions = {
		title: 'EVENTS',
		headerStyle: {
			backgroundColor: '#f0f3f8',
			borderBottomWidth: 0,
		},
		headerTintColor: '#5e5e82',
		headerTitleStyle: {
			fontFamily: 'RobotoCondensed-Bold',
		},
	};

	render() {
		return (
			<Container>
				<EventListView navigation={this.props.navigation}/>
			</Container>
		);
	}

}

export default EventList;