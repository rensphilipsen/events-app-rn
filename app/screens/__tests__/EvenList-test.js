// Copyright (c) 2014-present, Facebook, Inc. All rights reserved.

/**
 * Sample React Native Snapshot Test
 */

'use strict';

import 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import EventList from "../EventList";
import ShallowRenderer from 'react-test-renderer/shallow';

jest.setTimeout(15000);

it('renders correctly', () => {
	const renderer = new ShallowRenderer();
	renderer.render(<EventList/>);
	const tree = renderer.getRenderOutput();
	expect(tree).toMatchSnapshot();
});