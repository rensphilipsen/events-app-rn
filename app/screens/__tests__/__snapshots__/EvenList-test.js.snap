// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<_default>
  <Connect(EventListView) />
</_default>
`;
